submissions/blueprint-rant/blueprint-rant.md

```diff
@@ -0,0 +1,51 @@
+
+### Creativity in Factorio - Kovarex enrichment
+
+As a final note before my conclusion, I'd like to highlight just how much space for creativity there is in this game and the missed opportunities airising from blindly importing blueprints, using the [Kovarex enrichment process](https://wiki.factorio.com/Kovarex_enrichment_process). This late game uranium processing method allows you to increase the quantity of useful Uranium-235 using a cyclical process. I believe that it is probably one of the more blueprinted processes in the game due to the initial complexity it appears to have.
+
+![](Kovarex recipe from wiki)
+
+Recently on the [r/factorio](https://www.reddit.com/r/factorio/) subreddit, there was a flood of different Kovarex enrichment setup photos, each one trying to one up the last in creativity, efficiency and all out weirdness. I'd like to share a few of these here, to highlight just how many options there are in this game.
+
+https://www.reddit.com/r/factorio/comments/ju509t/my_noobish_try_at_kovarex_enrichment/
+
+https://www.reddit.com/r/factorio/comments/it53gn/so_i_built_a_kovarex_enrichment_process_setup/
+
+https://www.reddit.com/r/factorio/comments/jkmkyc/my_overkill_beaconed_buffered_uranium_processing/
+
+https://www.reddit.com/r/factorio/comments/hgb8zn/the_1_million_monkeys_method_of_kovarex_enrichment/
+
+https://www.reddit.com/r/factorio/comments/jj4nsl/my_take_on_kovarex_circle_nuketrain_violently/
+
+### Stop using blueprint websites
+
+The beauty of Factorio is of course how free you are to do whatever your heart desires, which is a both a blessing and a curse. The ability to use these simple building blocks to build sprawling factories and compact megabases is truely fantastic. Unfortunately, by providing and open sandbox, Wube has accidentally created a situation where people can "play the game" without truely playing it and not enjoying it along the way. The blueprint import feature will never be removed, so instead I implore you to design your own stuff, avoiding looking at other peoples designs as much as possible and be creative. I'm sure that if you take the time to design something, it will be much more rewarding than searching for blueprints for 20 minutes, trying to find exactly what you want.
```

Comment:
I don't think you `"play the game" without truly playing it and not enjoying it along the way` sounds a little patronising. They are enjoying the way they play, even if they're using blueprints. I think the note you want to hit here is more that it _could_ be possible to have *more* enjoyment. And maybe mention again that the complexity should not scare you into not trying to build it yourself. It's the best feeling seeing something you don't understand, and then a few hours later you have a factory producing it that you designed yourself.